__snapshots__/github-release.js

```diff
@@ -111,13 +111,13 @@ exports['GitHubRelease extractLatestReleaseNotes php-yoshi extracts appropriate
 
 `
 
-exports['GitHubRelease createRelease creates and labels release on GitHub 1'] = {
-  'tag_name': 'v1.0.3',
+exports['GitHubRelease createRelease creates releases for submodule in monorepo 1'] = {
```

Comment:
I'm having a hard time with the snapshot files. when I run `npm test` any change introduced re-writes all `'` -> `"` for the entire file. Here I've tried to hand curate the appropriate changes but I'd really like to understand what my issue is with the auto-writing.

Reply:
@joeldodge79 rather than manually updating the snapshot files, you can run:

```
npm run test:snap
```

and it will capture the current state of things.

Reply:
Even running `npm run test:snap` I still get the `'` -> `"` problem so the diff (see below) is totally useless :-(  I'm having a hard time tracking down why my environment wants to make these changes (something with prettier somehow?) any thoughts?

```diff
diff --git a/__snapshots__/commit-split.js b/__snapshots__/commit-split.js
index e8ec512..9e7e5ff 100644
--- a/__snapshots__/commit-split.js
+++ b/__snapshots__/commit-split.js
@@ -1,569 +1,569 @@
 exports['CommitSplit partitions commits based on path from root directory by default 1'] = {
-  'PubSub': [
+  "PubSub": [
     {
-      'sha': '0a8477108a26aeb21d7af06e62be4ae5cb00ad42',
-      'message': 'fix: Update PubSub timeouts. (#1967)',
-      'files': [
-        'PubSub/src/V1/resources/subscriber_client_config.json',
-        'PubSub/synth.metadata'
+      "sha": "0a8477108a26aeb21d7af06e62be4ae5cb00ad42",
+      "message": "fix: Update PubSub timeouts. (#1967)",
+      "files": [
+        "PubSub/src/V1/resources/subscriber_client_config.json",
+        "PubSub/synth.metadata"
       ]
     }
   ],
```